src/pages/CropSelector/CropSelector.js

```diff
@@ -156,8 +156,8 @@ const CropSelector = (props) => {
             cropData={cropData}
             activeCropData={cropDataRedux.filter((crop) => activeCropIdsRedux.includes(crop.id))}
             comparisonView={comparisonView}
-            toggleComparisonView={() => { setComparisonView(!comparisonView); }}
-            toggleListView={() => { setIsListView(!isListView); }}
+            setComparisonView={setComparisonView}
+            setIsListView={setIsListView}
```

Comment:
ok small nitpick. can you name them both either isComparisonView and isListView or listView and comparisonView?